### ISSUE TITLE
requests about more metadata on the variables

### ISSUE THREAD

**Issue by jerabaul29** [OpenDrift/trajan#100]
From the data management group:

"
If you do not have standard_name for variables, please always add long_name. "Unit " should be "units". I'm getting quite few errors when trying to validate (https://compliance.ioos.us/index.html), and I can't visualize in panoply. Are your data supposed to be visualised as trajectory? 
"

I guess this should be part of trajan so this is automatically done? :) I will try to add this in a PR :) .

**Comment by gauteh** [comment]
Related to: https://github.com/OpenDrift/trajan/issues/78

**Comment by knutfrode** [comment]
Yes, I corrected now attribute name `unit` to `units` at several places in the code:
https://github.com/OpenDrift/trajan/pull/101

Otherwise I believe standard_name variables should be correct for the core variables (e.g. `lontitude, latitude, time`), however seems to be missing for the more specialized variables e.g. in reader_omb (#78)

**Comment by gauteh** [comment]
I think a bunch of them have standard names, it is just that I wrote long_name rather than standard_name in the sfy code.

**Comment by gauteh** [comment]
the frequency bins should be a coordinate variable to the spectrum.
